ipatests/pytest_plugins/integration/tasks.py

```diff
@@ -1381,3 +1381,29 @@ def add_dns_zone(master, zone, skip_overlap_check=False,
                                     host.hostname + ".", '--a-rec', host.ip])
     else:
         logger.debug('Zone %s already added.', zone)
+
+
+def run_dnsserver_find(host, server_to_find, returncode=None):
+    """
+    Find DNS-server details
+    """
+    kinit_admin(host)
+    cmd = host.run_command(["ipa", "dnsserver-find", server_to_find],
+                           raiseonerr=False)
+    if cmd.returncode == returncode:
+        print cmd.stdout_text
+    else:
+        print cmd.stderr_text
+
+
+def run_server_find(host, server_to_find, returncode=None):
```

Comment:
This function has the same naming structure as other run_xx_xxx function but behaves differently which might confuse further reader or user. It is called run_server_find. One would expect that it runs it and returns output. But it fact it does a check to returncode and do nothing it even doesn't return the output. 

IMO it should either do some assertion but then should be named something like `def test_server_record_presence(host, server, should_exist). Or should return the output and be called `run_server_find`.